tests/sqlalchemy_dialect_compliance/test_dialect_compliance.py

```diff
@@ -244,29 +737,29 @@ def test_select_exists_false(self, connection):
 del LongNameBlowoutTest
 
 
-class SimpleUpdateDeleteTest(_SimpleUpdateDeleteTest):
-    """The base tests fail if operations return rows for some reason."""
+# class SimpleUpdateDeleteTest(_SimpleUpdateDeleteTest):
```

Comment:
IF we are ditching this test, then let's delete OR remove the comment octothorps if the test is still valid/needed.

Reply:
Will be removing this and cleaning up this file in #987